API/app/Http/Controllers/Api/GroceryItemController.php

```diff
@@ -63,15 +64,15 @@ public function store(Request $request)
     public function show(Request $request, $id)
     {
         $groceryItems = GroceryItem::where('shopping_list_id', $id)->get();
-        // Authorize user
-        $userId = $request->user_id;
         // This is how we get the user_id of the resource for the grocery item 
         // (i.e. check its corresponding shopping list)
         $shoppingList = ShoppingList::findOrFail($id);
+        // Grab user from sanctum
+        $user = Auth::user();
         
         // Sharing will be a feature added here later
```

Comment:
Remove this comment (sharing is already a feature that has been added to SpeedCart).